swagger/swagger.yaml

```diff
@@ -497,6 +497,12 @@ paths:
           required: false
           type: boolean
           default: true
+        - name: includeSubClasses
+          in: query
```

Comment:
I'll also update this once we finalize the implementation.

Reply:
We don't need this, you can remove.